Owl/app/src/main/AndroidManifest.xml

```diff
@@ -32,6 +32,7 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <activity android:name="androidx.activity.ComponentActivity" />
```

Comment:
Is this for tests? Does this have to be in the main src set?

Reply:
Ah forgot to move